### ISSUE TITLE
hilite.me Site Would Benefit with Update to latest Pygments

### ISSUE THREAD

**Issue by Olliebrown** [alexkay/hilite.me#10]
I make use of the hilite.me website frequently for preparing code snippets to post to my course websites and find it quite useful!  However, I am now finding that it is out of date with the latest JavaScript languages features and this seems to be because it is built against an earlier version of Pygments.

Specifically, ES6 string-templates (which are pretty ubiquitous these days and handy) do not highlight correctly with the web site.  I did a local build of the code here on github and ran the dev server and it looks like the latest version of Pygments does support this syntax (and your code picks up those newer features without needing any changes)!  It also picked up several new color schemes provided by Pygments without changes (very nice)!

I'm happy to just keep running it locally when I need it but I just thought I would bring this to your attention in case it's easy to update the hilite.me website.  There could, in theory, be lots of other small language changes like this that are being missed just because it's built against and older version of Pygments.

## Current hilite.me site:
![Example from hilite.me](https://preview.ibb.co/j8kPA7/hiliteme_current.png)

## My local dev build:
![Example from dev build](https://preview.ibb.co/fkdNxn/hiliteme_dev.png)

**Comment by christiansestier** [comment]
Yes, thank you for this really good tool. Just following up on Olliebrown comment. Are there any plans to update the hilite.me api with the latest version of Pygments?

**Comment by mesibo** [comment]
We have updated to use Python 3 and the latest Pygments. I hope you find it useful. 

Demo https://pygments-web-interface.onrender.com/

GitHub https://github.com/mesibo/pygments-web-interface

**Comment by alexkay** [comment]
I updated hilite.me to use the latest Pygments. As part of the update it website has been moved to a new host, it might take a few hours for DNS changes to propagete.
